### DISCUSSION TITLE
Unifi Express support?

### DISCUSSION THREAD

**Discussion by davidvd102** [SierraSoftworks/tailscale-udm#97] [Q&A] [answered]
I tried with Unifi Express on OS version: 4.0.9.16908 and it failed when curl, look below

`root@UniFi-Express:~# curl -sSLq https://raw.github.com/SierraSoftworks/tailscale-udm/main/install.sh | sh
curl: (60) SSL certificate problem: self signed certificate
More details here: https://curl.se/docs/sslcerts.html

curl failed to verify the legitimacy of the server and therefore could not
establish a secure connection to it. To learn more about this situation and
how to fix it, please visit the web page mentioned above.
Installing latest Tailscale package repository...
curl: (60) SSL certificate problem: self signed certificate
More details here: https://curl.se/docs/sslcerts.html

curl failed to verify the legitimacy of the server and therefore could not
establish a secure connection to it. To learn more about this situation and
how to fix it, please visit the web page mentioned above.
curl: (60) SSL certificate problem: self signed certificate
More details here: https://curl.se/docs/sslcerts.html

curl failed to verify the legitimacy of the server and therefore could not
establish a secure connection to it. To learn more about this situation and
how to fix it, please visit the web page mentioned above.
Updating package lists...
Hit:1 https://security.debian.org/debian-security bullseye-security InRelease
Hit:2 https://deb.debian.org/debian bullseye InRelease
Hit:3 https://deb.debian.org/debian bullseye-updates InRelease
Hit:4 https://deb.debian.org/debian bullseye-backports InRelease
Hit:5 https://apt.artifacts.ui.com bullseye InRelease
Reading package lists... Done
Building dependency tree... Done
Reading state information... Done
39 packages can be upgraded. Run 'apt list --upgradable' to see them.
Installing Tailscale ...
Reading package lists... Done
Building dependency tree... Done
Reading state information... Done
E: Unable to locate package tailscale`

**Comment by davidvd102** [comment]
I tried -k and --insecure on curl but keep throwing same error

**Comment by bslatyer** [comment, marked as answer]
@davidvd102, yes, it is supported.
